### DISCUSSION TITLE
How to select preferred currency for subscription?

### DISCUSSION THREAD

**Discussion by ModalityZ** [mdn/mdn-community#670] [MDN Plus]
I have been wanting to support MDN for years, and I'm so glad to see there's a way now.

I would like to pay in USD but the amount due is shown in EUR presumably due to geofencing (I'm in Europe right now). How can I select my currency to avoid exchange fees?

**Comment by dipikabh** [comment]
Hi @ModalityZ, thanks for your support and enthusiasm for MDN!

Unfortunately, the system is designed with geofencing restrictions, which means it recognizes and processes transactions based on specific countries and currencies. As a result, it only accepts payments that match these predefined settings, and currently, there is no way to bypass or alter these limitations.

We appreciate your understanding and are grateful to have you as part of the MDN community.
